WBooks/src/app/components/Header/index.js

```diff
@@ -1,17 +1,20 @@
 import React from 'react';
 import { Text, ImageBackground } from 'react-native';
-import nabBarImage from  '../../assets/bc_nav_bar.png';
+
+import nabBarImage from '../../assets/bc_nav_bar.png';
```

Comment:
`navBarImage` ?

Reply:
yes... it's the image I was using in the header... I will use it again soon :p

Reply:
sorry, i meant the name for `nav` instead of `nab`  jaj